00004-Vulnerability-disclosure-and-embargo-policy/README.md

```diff
@@ -0,0 +1,252 @@
+the list of security advisories and fixes.
+
+The two pages are "Vulnerability Reporting and Embargo Policy" and "Enarx 
+Security Advisories."
+
+--- 
+### Vulnerability Reporting and Embargo Policy
+The Enarx project aims to be as safe to use as possible and therefore to 
+produce code that is as secure as possible. As such, we welcome any and all 
+research into Enarx and appreciate your efforts to take the time to 
+responsibly disclose any issues you find.
+
+Vulnerabilities may be discovered in:
+1. Enarx code
+2. Enarx dependencies (including compilers)
+3. Enarx documentation
+4. 3rd party hardware or firmware
+
+All security bugs in Enarx should be reported to the security team. 
+To do so, please reach out either by email at security@enarx.dev or on our
+[chat](https://chat.enarx.dev) stating you'd like to report a security
+issue, but **without divulging details**.
+
+We treat both email and chat as a public channels, so simply saying that
+you’re trying to get a hold of someone from the security team is fine.
+
+From then, we will establish a more secure area where the issue can be
+discussed, in the form of a 
+[Github Security Advisory](https://help.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories).
+
+You will be invited to join this private area to discuss specifics. Doing so 
+allows us to start with a high level of confidentiality and relax it if the 
+issue is less critical, moving to work on the fix in the open.
+
+This method provides several other advantages:
+1. Avoids email, which is unfortunately very difficult to use securely
+2. Enables private collaboration to fix the vulnerability in 
+a temporary private fork
+3. Facilitates drafting a security advisory and using the draft to 
+privately discuss the impact of the vulnerability on the project
+4. Simplifies publishing the security advisory and alerting the 
+community of the vulnerability
+
+Your initial contact will be acknowledged within 48 hours, and you’ll receive 
+a more detailed response within 96 hours indicating the next steps in handling 
+your report.
+
+After the initial reply to your report, the security team  will endeavor to 
+keep you informed of the progress being made towards a fix and full 
+announcement. As recommended by 
+[RFPolicy](https://dl.packetstormsecurity.net/papers/general/rfpolicy-2.0.txt),
+ these updates will be sent at least every five working days.
+
+If you have not received a reply to your initial contact within 96 hours, or
+have not heard from the security team for the past five days, there are a few
+steps you can take (in suggested order):
+- Contact the current security coordinator (Nathaniel McCallum, aka
+[npmccallum](https://github.com/npmccallum)) directly
+- Contact the back-up contact (Mike Bursell, aka 
+[MikeCamel](https://github.com/mikecamel)) directly  
+  In both these cases, you can reach out
+  - by email
```

Comment:
Perhaps this is intended, but Mike's email is not discoverable through this document. (Nathaniel's is in his GitHub bio.)

Reply:
Ah, good catch. @MikeCamel, how do you want to deal with this?

Reply:
Added to my github bio.